workspaces/desktop/server/server.js

```diff
@@ -11,7 +11,7 @@ const app = express();
 express_ws(app);
 const server = http.createServer(app);
 const ws_proxy = http_proxy.createProxyServer({
-  target: 'http://localhost:5901',
+  target: 'http://127.0.0.1:5901',
```

Comment:
This was necessary to make the image work again. Without this, it tries to use IPv6, which empirically doesn't work. The logs imply it won't either:

```
2025-04-17T18:14:53.164336547Z [stderr] /usr/bin/x11vnc: 17/04/2025 18:14:53 Autoprobing selected TCP6 port 5900
2025-04-17T18:14:53.164337672Z 17/04/2025 18:14:53 listen6: bind: Address already in use
2025-04-17T18:14:53.164338756Z 17/04/2025 18:14:53 Not listening on IPv6 interface.
```

I spent approximately zero time looking into this, but I can confirm that the image now boots and works correctly.

Reply:
Not the first time this was an issue (see #10006)...